spot-client/src/common/remote-control/XmppConnection.test.js

```diff
@@ -58,11 +58,8 @@ describe('XmppConnection', () => {
                 .toEqual({
                     from: IQ_FROM,
                     localUpdate: false,
-                    state: {
-                        isSpot: true,
-                        videoMuted: true
-                    },
-                    type: 'unavailable'
+                    type: 'unavailable',
```

Comment:
can you add extra special case for unavailable instead of removing a regular one?

Reply:
nevermind I got confused by reading commit after commit.